platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -111,13 +111,16 @@ public void init() throws Exception {
             migrationWarnings.addAll(systemConfigurationMigration.export(exportDirectory));
             migrationWarnings.addAll(exportMigratables(exportDirectory));
         } catch (IOException e) {
-            String message = "Unable to create export directories.";
-            LOGGER.error(message, e);
-            throw new MigrationException(message, e);
+            LOGGER.error("Unable to create export directories", e);
+            throw new MigrationException(String.format(
+                    "Failed to export configurations: Unable to create export directories. %s",
```

Comment:
Same as in `MigrationUtil`.  It looks like we're repeating `"Failed to export configurations:"` a lot.